OpenRA.Mods.Common/Activities/Air/TakeOff.cs

```diff
@@ -43,10 +52,10 @@ public override Activity Tick(Actor self)
 			var destination = rp != null ? rp.Location :
 				(hasHost ? self.World.Map.CellContaining(host.CenterPosition) : self.Location);
 
-			if (NextInQueue == null)
+			if (moveToRallyPoint() || NextActivity == null)
```

Comment:
Right now there is no way for callers to specify that it should *never* go to the rally point.  Can we move this `NextActivity == null` check into the default for `moveToRallyPoint` instead?

Reply:
I guess we can. It would make sense for parents to then be able to check whether the TakeOff activity had anything queued. Wouldn't need it for this PR in particular, but other places might. Eventually. Maybe. ;)